pallets/pallet-bonded-coins/src/curves.rs

```diff
@@ -0,0 +1,253 @@
+use core::ops::{AddAssign, BitOrAssign, ShlAssign};
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_arithmetic::ArithmeticError;
+use substrate_fixed::{
+	traits::{Fixed, FixedSigned, ToFixed},
+	transcendental::{exp, ln, sqrt},
+	types::I9F23,
+};
+
+#[derive(Clone, Debug, Encode, Decode, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+pub enum Curve<F> {
+	PolynomialFunction(PolynomialFunctionParameters<F>),
+	SquareRootBondingFunction(SquareRootFunctionParameters<F>),
+	LMSR(LMSRFunctionParameters<F>),
+}
+
+pub enum Operation<F> {
+	Mint(Vec<F>),
```

Comment:
Added passive supply into the enum: https://github.com/KILTprotocol/kilt-node/pull/757/files/4a4174cbc0e795c208551efdfd7f32d1c8cfcbdc#r1808202500

Reply:
[7f3bf57](https://github.com/KILTprotocol/kilt-node/pull/765/commits/7f3bf57e299d5ef09219b9805ff5bdb18744b077)